docs/source/user_guide/timeseries.ipynb

```diff
@@ -851,7 +851,128 @@
   },
   {
    "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Forecasting into the future\n",
+    "\n",
+    "Our previous examples of shown using a pipeline to predict data we had at training time. However, we can also use EvalML time series pipelines to forecast dates into the future as long we provide data that meets the requirements of **Predicting on unseen data** as well.\n",
```

Comment:
Typo: "examples of shown" -> "examples have shown". Also maybe "to predict data we had at training time" -> "to predict *on* data we had at training time"